tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -999,22 +999,6 @@ public function testActivityCreate() {
     $this->assertEquals("$ 200.00 - STUDENT", $activity->subject, 'Check for total amount in activity.');
   }
 
-  /**
-   * Test checkContributeSettings.
-   */
-  public function testCheckContributeSettings() {
-    $settings = CRM_Contribute_BAO_Contribution::checkContributeSettings('deferred_revenue_enabled');
```

Comment:
@eileenmcnaughton is there any handling here for storing the settings in the correct way if we get in the wrong format or saying its in the wrong format?

Reply:
@seamuslee001 how do you mean? It just copies the setting from the wrong place (nested in a key) to the right place (it actually leaves a duplicate but I think that's OK since we can clean that up when the rest of the settings are migrated from that key)